src/SFA.DAS.EAS.Web/Views/EmployerAccountTransactions/Index.cshtml

```diff
@@ -1,239 +1,68 @@
-                        </tr>
-                                }
-                </tbody>
-            </table>
-                                }
-
-        <ul class="previous-next-navigation">
-            @if (!Model.Data.IsFirstMonthOfLevy && Model.Data.AccountHasPreviousTransactions)
-            {
-                <li class="previous">
-                    <a href="@Url.Action("Index")/@previousMonthDate.Year/@previousMonthDate.Month">
-                        Previous <span class="hide-mobile">month</span>
-                        <span class="detail">@previousMonthDate.ToGdsFormatWithoutDay()</span>
-                    </a>
-                </li>
-            }
-            @if (!Model.Data.IsLatestMonth)
-            {
-                <li class="next">
-                    <a href="@Url.Action("Index")/@nextMonthDate.Year/@nextMonthDate.Month">
-                        Next month
-                        <span class="detail">@nextMonthDate.ToGdsFormatWithoutDay()</span>
-
-                    </a>
-                </li>
-            }
-        </ul>
+    <div class="column-half">
+        <div class="data-wrap">
+            <div class="data">
+                <dl>
+                    <dt>
+                        Current levy funds
+                    </dt>
+                    @{
+                        var balance = Model.Data.CurrentLevyFunds.ToString("C0", culture);
+                        var balanceClass = string.Empty;
+
+                        if (balance.Length > 8)
+                        {
+                            balanceClass = "long";
+                        }
+                        if (balance.Length > 10)
+                        {
+                            balanceClass = "longer";
+                        }
 
-        <details class="feedback">
-            <summary><span class="summary">Help us improve this section</span></summary>
-            <div class="survey">
-            <script id="ss-embed-396029">
-                (function (d, w) {
-                    var s, ss; ss = d.createElement('script');
-                    ss.type = 'text/javascript';
-                    ss.async = true;
-                    ss.src = ('https:' == d.location.protocol ? 'https://' : 'http://') + 'www.smartsurvey.co.uk/s/r/embed.aspx?i=350750&c=396029';
-                    s = d.getElementsByTagName('script')[0];
-                    s.parentNode.insertBefore(ss, s);
-                })(document, window);
-            </script>
+                    }
+                    <dd class="@balanceClass">@balance</dd>
+                </dl>
             </div>
-        </details>
-
-       
+        </div>
+    </div>
+</div>
+<div class="grid-row">
+    <div class="column-half">
+        <h2 class="heading-medium">
+            <a title="View transactions" href="@Url.Action("TransactionsView", "EmployerAccountTransactions", new {HashedAccountId = Model.Data.Account.HashedId, month = 0, year = 0})">
```

Comment:
This will likely append a `/0/0` path to the URL, is there an alternative?

Reply:
if you leave them off I think it will use the default for whatever type that is on that action (probably int so 0)

Reply:
This does append a /0/0 that's, the "Your transaction" view requires this.

Reply:
we can use new `{HashedAccountId = Model.Data.Account.HashedId, month = DateTime.Now.Month, year = DateTime.Now.Year};` instead of 0/0 so application will redirects the page to the current month in current year